.rubocop.yml

```diff
@@ -0,0 +1,20 @@
+AlignParameters:
+  Enabled: false
+
+Encoding:
+  Enabled: false
+
+LineLength:
+  Max: 200
+
+HashSyntax:
+  EnforcedStyle: hash_rockets
```

Comment:
I don't agree, mostly because I like the newer style, and we require ruby v2 for stuff nowadays, so I don't think there's any reason to enforce it.